tests/Fixtures/Integration/misc/property_promotion_with_nullable_type.test

```diff
@@ -0,0 +1,20 @@
+--TEST--
+Integration test to verify that property promotion with nullable type does not crash the Tokenizer.
+--RULESET--
+{
+    "ternary_to_elvis_operator": true
```

Comment:
random rule. not related to the problem...
most probably the test case can be moved directly to `TokenizerTest` class